app/controllers/planners_controller.rb

```diff
@@ -6,7 +6,7 @@ def index
 
   def show
     @planner = Planner.find(params[:id])
-    @reservation_frames = ReservationFrame.after_current_time.status_ok.sort_reserved_at_asc
+    @reservation_frames = ReservationFrame.where(planner_id: params[:id]).after_current_time.status_ok.sort_reserved_at_asc
```

Comment:
上記で`@planner`は定義してるから

```suggestion
    @reservation_frames = @planner.reservation_frames.after_current_time.status_ok.sort_reserved_at_asc
```

これでいけるはず